default.nix

```diff
@@ -6,72 +6,126 @@ with pkgs;
+    "${tezos-static-mainnet}/bin";
+  babylonnet-binaries =
+    packDirectory "tezos-babylonnet-binaries-${babylonnet.rev}"
+    "${tezos-static-babylonnet}/bin";
+
+  mkPackageDescs = { executableName, binPath, description }@executableDesc:
+    let
+      mainnetDesc = {
+        inherit description;
+        bin = "${tezos-static-mainnet}/${binPath}";
+        gitRevision = mainnet.rev;
+        project = "${executableName}-mainnet";
+        version = toString timestamp;
+        arch = "amd64";
+        license = "MPL-2.0";
+        dependencies = "";
+        maintainer = "Serokell https://serokell.io";
+        licenseFile = "${root}/LICENSES/MPL-2.0.txt";
+        branchName = "mainnet";
+      };
+      babylonnetDesc = mainnetDesc // {
+        bin = "${tezos-static-babylonnet}/${binPath}";
+        project = "${executableName}-babylonnet";
+        gitRevision = babylonnet.rev;
+        branchName = "babylonnet";
+      };
+    in [ mainnetDesc babylonnetDesc ];
+
+  tezos-executables = [
+    { executableName = "tezos-client";
+      binPath = "/bin/tezos-client";
+      description = "CLI client for interacting with tezos blockchain";
+    }
+    {
+      executableName = "tezos-admin-client";
+      binPath = "/bin/tezos-admin-client";
+      description = "Administration tool for the node";
+    }
+    { executableName = "tezos-node";
+      binPath = "/bin/tezos-client";
+      description =
+        "Entry point for initializing, configuring and running a Tezos node";
+    }
+    { executableName = "tezos-baker";
+      binPath = "/bin/tezos-baker-${mainnet.binarySuffix}";
+      description = "Daemon for baking";
+    }
+    { executableName = "tezos-accuser";
+      binPath = "/bin/tezos-accuser-${mainnet.binarySuffix}";
+      description = "Daemon for accusing";
+    }
+    { executableName = "tezos-endorser";
+      binPath = "/bin/tezos-endorser-${mainnet.binarySuffix}";
+      description = "Daemon for endorsing";
+    }
+    { executableName = "tezos-signer";
+      binPath = "/bin/tezos-signer";
+      description = "A client to remotely sign operations or blocks";
+    }
+  ];
+
+  packageDescs = lib.flatten (map mkPackageDescs tezos-executables);
 
   buildDeb = import ./packageDeb.nix { inherit stdenv writeTextFile dpkg; };
   buildRpm = packageDesc:
-  import ./packageRpm.nix { inherit stdenv writeTextFile gnutar rpm buildFHSUserEnv; }
-    (packageDesc // { arch = "x86_64"; });
-
-  mainnet-rpm-package = buildRpm packageDesc-mainnet;
+    import ./packageRpm.nix {
+      inherit stdenv writeTextFile gnutar rpm buildFHSUserEnv;
+    } (packageDesc // { arch = "x86_64"; });
 
-  mainnet-deb-package = buildDeb packageDesc-mainnet;
+  moveDerivations = name: drvs:
+    stdenv.mkDerivation rec {
+      inherit name drvs;
 
-  babylonnet-rpm-package = buildRpm packageDesc-babylonnet;
+      buildCommand = ''
+        mkdir -p ${name}
+        backup=$PWD
+        for drv in $drvs; do
+          cd $drv
```

Comment:
why all the cd's instead of absolute paths?

Reply:
AFAIR, then there is going to be this `/nix/store/<hash>` prefixes

Reply:
ah, lol, I'm idiot